targets/TARGET_NORDIC/TARGET_NRF5/TARGET_MCU_NRF51822_UNIFIED/sdk_patch/sdk_config.h

```diff
@@ -6,8 +6,10 @@
 #define UART_DEFAULT_CONFIG_BAUDRATE UART0_CONFIG_BAUDRATE
 #define UART_DEFAULT_CONFIG_PARITY   UART0_CONFIG_PARITY
 #define UART_DEFAULT_CONFIG_HWFC     UART0_CONFIG_HWFC
-#define CTS_PIN_NUMBER               UART0_CONFIG_PSEL_CTS
-#define RTS_PIN_NUMBER               UART0_CONFIG_PSEL_RTS
+
+/* Use config at PinNames.h file in TARGET_XXXX directory */
```

Comment:
@0xc0170 Yes, removing the CTS_PIN_NUMBER/RTS_PIN_NUMBER defines from sdk_config.h would fix it for us.